test/nonregression/iotools/test_run_converters.py

```diff
@@ -206,6 +207,32 @@ def run_ukbtobids(input_dir: PathLike, output_dir: PathLike, ref_dir: PathLike)
     compare_folders(output_dir / "bids", ref_dir / "bids", output_dir / "bids")
 
 
+def run_genfitobids(
+    input_dir: PathLike, output_dir: PathLike, ref_dir: PathLike
+) -> None:
+    from pathlib import PurePath
+
+    from clinica.iotools.converters.genfi_to_bids.genfi_to_bids import convert_images
+    from clinica.utils.check_dependency import check_dcm2niix
+
+    # Convert
+    input_dir = PurePath(input_dir)
+    output_dir = PurePath(output_dir)
+    ref_dir = PurePath(ref_dir)
+
+    # Act
+    check_dcm2niix()
+    convert_images(
+        input_dir / "unorganized",
+        output_dir / "bids",
+        path_to_clinica=None,
```

Comment:
Looks like I'm tired... 😓 

```suggestion
        path_to_clinical=None,
```